includes/core/class-mail.php

```diff
@@ -384,9 +384,21 @@ function prepare_template( $slug, $args = array() ) {
 			add_filter( 'um_template_tags_replaces_hook', array( &$this, 'add_replace_placeholder' ), 10, 1 );
 
 			// Convert tags in email template
-			return um_convert_tags( $message, $args );
-		}
+			$message = um_convert_tags( $message, $args );
 
+			// Strip tags in plain text email.
+			// Important! Don't use HTML in plain text emails.
+			if ( ! UM()->options()->get( 'email_html' ) ) {
+				$message_nl = str_replace(
```

Comment:
Because there can be not only `<p>` or `<div>` tags where we have to add \n for plain-text formatting